clang-tools-extra/clang-tidy/readability/ContainerContainsCheck.cpp

```diff
@@ -51,6 +53,12 @@ void ContainerContainsCheck::registerMatchers(MatchFinder *Finder) {
   const auto Literal0 = integerLiteral(equals(0));
   const auto Literal1 = integerLiteral(equals(1));
 
+  const auto StringLikeClass = cxxRecordDecl(
+      hasAnyName("::std::basic_string", "::std::basic_string_view",
+                 "::absl::string_view"));
```

Comment:
I'm a fan of not hard-coding any classes when possible. IMO it's fine to match `npos` without checking the DeclContext, it's not a common name.

Reply:
Agree!